app/views/teachers/components/TestStudentModal.vue

```diff
@@ -71,16 +73,18 @@ export default Vue.extend({
   },
   created () {
     this.classLoading = true
-    fetchJson(`/db/classroom?ownerID=${me.id}&project=code,name,ownerID`)
+    Api.fetchByOwner(me.id, { project: ['code', 'name', 'ownerID'] })
       .then(data => {
         this.classLoading = false
         this.classrooms = data
-        this.classCode = data[0].code
+        this.classCode = data?.[0].code
       })
   },
 
   methods: {
     joinClassroom () {
+      const text = 'Switching to test student account..'
```

Comment:
should probably i18n this string at both places

Reply:
haha good catch, let's do it